src/animation/frontend/count/index.js

```diff
@@ -22,7 +22,10 @@ const speedConfig = {
  * @return Configuration options.
  */
 const getConfiguration = ( elem ) => {
-	const parent = elem.parentElement;
+	let parent = elem.parentElement;
+	if (parent && (['STRONG', 'B'].includes(parent.tagName))) {
```

Comment:
I suspect that this problem can also be in italic format. Can you do a quick check to see if it is **only** for bold format?

Reply:
@Soare-Robert-Daniel I've checked it with italic formatting and noticed the same issue. I also tested with other formatting tags, and the problem persists. I've fixed it in the latest commit. Could you please check and let me know if any changes are needed?